app/models/grda_warehouse/data_source.rb

```diff
@@ -31,7 +31,8 @@ class GrdaWarehouse::DataSource < GrdaWarehouseBase
   accepts_nested_attributes_for :projects
   has_many :exports, class_name: 'GrdaWarehouse::Hud::Export', inverse_of: :data_source
   has_many :group_viewable_entities, -> { where(entity_type: 'GrdaWarehouse::DataSource') }, class_name: 'GrdaWarehouse::GroupViewableEntity', foreign_key: :entity_id
-
+  # FIXME extension
+  has_many :custom_data_element_definitions, class_name: 'Hmis::Hud::CustomDataElementDefinition'
```

Comment:
I couldn't for the life of me get this working as an extension. The attempt was:


`drivers/hmis/extensions/grda_warehouse/data_source_extension.rb`

```
module Hmis
  module GrdaWarehouse
    module DataSourceExtension
      extend ActiveSupport::Concern

      included do
        has_many :custom_data_element_definitions, class_name: '::Hmis::Hud::CustomDataElementDefinition'
      end
    end
  end
end

```

And the error was something like "can't resolve `Hmi`" ...